Index: providers/SupabaseProvider.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>'use client'\n\nimport {Database} from \"@/typs_db\";\nimport React, {useState} from \"react\";\nimport {createClientComponentClient} from \"@supabase/auth-helpers-nextjs\";\nimport {SessionContextProvider} from \"@supabase/auth-helpers-react\";\n\ninterface SupabaseProviderProps {\n    children: React.ReactNode\n}\n\nconst SupabaseProvider: React.FC<SupabaseProviderProps> = ({children}) => {\n    const [supabaseClient] = useState(() =>\n        createClientComponentClient<Database>()\n    )\n\n    return (\n        <SessionContextProvider supabaseClient={supabaseClient}>\n            {children}\n        </SessionContextProvider>\n    )\n}\n\nexport default SupabaseProvider;
===================================================================
diff --git a/providers/SupabaseProvider.tsx b/providers/SupabaseProvider.tsx
--- a/providers/SupabaseProvider.tsx	
+++ b/providers/SupabaseProvider.tsx	
@@ -1,24 +1,24 @@
-'use client'
+"use client";
 
-import {Database} from "@/typs_db";
-import React, {useState} from "react";
-import {createClientComponentClient} from "@supabase/auth-helpers-nextjs";
-import {SessionContextProvider} from "@supabase/auth-helpers-react";
+import { useState } from "react";
+import { createClientComponentClient } from "@supabase/auth-helpers-nextjs";
+import { SessionContextProvider } from "@supabase/auth-helpers-react";
+import {Database} from "@/typs_db";
 
 interface SupabaseProviderProps {
-    children: React.ReactNode
-}
+    children: React.ReactNode;
+};
 
 const SupabaseProvider: React.FC<SupabaseProviderProps> = ({children}) => {
     const [supabaseClient] = useState(() =>
         createClientComponentClient<Database>()
-    )
+    );
 
     return (
         <SessionContextProvider supabaseClient={supabaseClient}>
             {children}
         </SessionContextProvider>
-    )
+    );
 }
 
 export default SupabaseProvider;
\ No newline at end of file
